vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go

```diff
@@ -29,7 +29,13 @@ var profileDirectory = "/etc/apparmor.d"
 // for the existence of the `apparmor_parser` binary, which will be required to
 // apply profiles.
 func IsEnabled() bool {
-	return supported.NewAppArmorVerifier().IsSupported() == nil
+	if err := supported.NewAppArmorVerifier().IsSupported(); err != nil {
+		if errors.Is(err, supported.ErrRootlessNotSupported) {
+			return true
```

Comment:
this may not be true though as a node that doesn't have apparmor but is rootless would return true in this case.  Is there a way we can verify support without rootless?

Reply:
Good point @haircommander . I've reworked this part but at the same I realized that I was in the wrong directory and used verndor directory instead of the actual common. Let's continue the discussion in https://github.com/containers/common/pull/2404. Thanks.